src/python_testing/TC_DeviceBasicComposition.py

```diff
@@ -941,6 +942,259 @@ def log_structured_data(start_tag: str, dump_string):
         log_structured_data('==== json: ', json_str)
         log_structured_data('==== txt: ', txt_str)
 
+    @async_test_body
+    async def test_TC_DESC_2_1(self):
+        asserts.assert_true('PIXIT.DESC.AGGREVISION' in self.matter_test_config.global_test_params,
```

Comment:
Is this typical to ask for a revision number as a parameter? why is this not from the spec?